pages/LoginPage.ts

```diff
@@ -0,0 +1,26 @@
+import { Page, Locator} from 'playwright/test';
+const { WellcomePage } = require('../pages/WellcomePage');
+
+class LoginPage {
+    page: Page;
+    user: Locator;
+    password: Locator;
+    loginButton: Locator;
+
+    constructor(page){
+        this.page = page;
+        this.user = page.getByPlaceholder('Enter your email...');
+        this.password = page.getByPlaceholder('Enter your password...');
+        this.loginButton = page.getByRole('button', { name: 'Log in' });
+    }
```

Comment:
I think we can use IDs instead of the PlayHolder text for user and password. I mean, this is working but this text could change. what do you think?

Reply:
You are right!, Done :)